src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionCoordinator.swift

```diff
@@ -93,7 +93,15 @@ final class ExposureDetectionCoordinator {
 	private let surveyURLProvider: SurveyURLProviding
 	
 	private func showHygieneRulesInfoScreen() {
-		// TODODODO: Navigate
+		let infoNavigationController = UINavigationController()
+		let viewController = HygieneRulesInfoViewController(
+			dismiss: {
+				infoNavigationController.dismiss(animated: true)
+			}
+		)
+		infoNavigationController.navigationBar.prefersLargeTitles = true
+		infoNavigationController.pushViewController(viewController, animated: false)
```

Comment:
```suggestion
		let viewController = HygieneRulesInfoViewController(
			dismiss: {
				infoNavigationController.dismiss(animated: true)
			}
		)
		let infoNavigationController = UINavigationController(rootViewController: viewController)
		infoNavigationController.navigationBar.prefersLargeTitles = true
```
pushViewController is not needed if init with a rootViewController

Reply:
🤓 imo not possible, because infoNavigationController is used before initialization. Does that compile on your machine?

Reply:
```suggestion
		let viewController = HygieneRulesInfoViewController(
			dismiss: { [weak self] in
				self?.navigationController?.dismiss(animated: true)
			}
		)
		let infoNavigationController = UINavigationController(rootViewController: viewController)
		infoNavigationController.navigationBar.prefersLargeTitles = true
		navigationController?.present(infoNavigationController, animated: true)
```
That way it should work - I don't think you need to capture `infoNavigationController` for dismiss. If it gets presented by `navigationController` use it for dismiss too?

Reply:
Done